ipapython/ipautil.py

```diff
@@ -245,6 +245,24 @@ def set_ip_net(self, ifnet):
         self._net = ifnet
 
 
+class CheckedIPAddressLoopback(CheckedIPAddress):
+    """IPv4 or IPv6 address with additional constraints with
+    possibility to use a loopback IP.
+    Reserved or link-local addresses are never accepted.
+    """
+    def __init__(self, addr, parse_netmask=True, allow_multicast=False):
+
+        super(CheckedIPAddressLoopback, self).__init__(
+                addr, parse_netmask=parse_netmask,
+                allow_multicast=allow_multicast,
+                allow_loopback=True)
+
+        if self.is_loopback():
+            # print is being used instead of a logger, because at this
+            # moment, in execution process, there is no logger configured
+            print("You are using a loopback IP: {}".format(addr))
```

Comment:
Should be at least printed to stderr. The bad thing is it's not going to appear in our logs but I guess we should be able to spot a loopback IP ourselves.

Reply:
done